app-admin/ngxtop/ngxtop-0.0.3_pre141201-r3.ebuild

```diff
@@ -16,12 +17,13 @@ SRC_URI="https://dev.gentoo.org/~jlec/distfiles/${P}.tar.xz"
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+RESTRICT="!test? ( test )"
 
-RDEPEND="
-	dev-python/docopt[${PYTHON_USEDEP}]
+RDEPEND="dev-python/docopt[${PYTHON_USEDEP}]
 	dev-python/pyparsing[${PYTHON_USEDEP}]
 	dev-python/tabulate[${PYTHON_USEDEP}]"
-DEPEND="${RDEPEND}"
+DEPEND="test? ( ${RDEPEND} )"
```

Comment:
`distutils_enable_tests` already does that.

Reply:
Thanks @mgorny, updated the PR.